src/Symfony/Contracts/EventDispatcher/EventListenerInterface.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Contracts\EventDispatcher;
+
+/**
+ * Marker interface for event listeners.
+ *
+ * The `__invoke` method is required and the `$event` argument have to be Type-Hinted with the listened event's class
+ * in order to be auto-configured.
```

Comment:
I'd suggest removing that part - the contracts don't know about autoconfiguration